### DISCUSSION TITLE
Private labels or tags for an issue or private issue

### DISCUSSION THREAD

**Discussion by gparant** [community/community#11651] [Projects and Issues]
Hello,

My need will be having a private issue (I know what there are discussion opened for that) with a private label or tag. I have labels or tags like "bug, feature..." but for the project I need to have more private like story point or other information that I could only share with my team. I don't see that feedback so I start this discussion 🚀 

Thanks 💪

**Comment by Alexander01998** [comment]
This would be especially useful for controversial labels like "invalid" or "won't fix".

Having these labels publicly causes way too many conflicts, but not having them can make it difficult to find the issues that you actually care about.

**Comment by kingthorin** [comment]
Vote it up over here: https://github.com/github/feedback/discussions/14474

We need a unified front :grinning:
